requirements_dev.txt

```diff
@@ -0,0 +1,47 @@
+amqp==2.5.0    
```

Comment:
This file we can generate later with correct values. You can leave it empty for now